metatron-app/metatron-discovery
# #2681 add engine monitoring datasource information

### Description

Add engine monitoring datasource information

**Related Issue** : #2681 



### How Has This Been Tested?

1. Go to Engine Monitoring
2. Go to datasource tab
3. Check the list and detail view

#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
